vault/diagnose/output.go

```diff
@@ -310,62 +273,107 @@ func (t *TelemetryCollector) getOrBuildResult(id trace.SpanID) *Result {
 	return r
 }
 
+func findAttributes(e trace.Event, attr1, attr2 attribute.Key) (string, string) {
+	var av1, av2 string
```

Comment:
why are these called `av1` and `av2`?

Reply:
attribute value 1 and 2.  This function is generically grabbing up two two attribute values given their keys.